commit 214e208d8018bf3bf4dad64afdd94393c1655ef4
Author: blaism <blaism>
Date:   Tue Apr 27 16:53:05 2010 +0000

    Changed default cipher suite to 1 instead of 3 for iol20

diff --git a/lib/ipmi_main.c b/lib/ipmi_main.c
index 3d93736..1a065eb 100644
--- a/lib/ipmi_main.c
+++ b/lib/ipmi_main.c
@@ -379,7 +379,7 @@ ipmi_main(int argc, char ** argv,
 	char * seloem   = NULL;
 	int port = 0;
 	int devnum = 0;
-	int cipher_suite_id = 3; /* See table 22-19 of the IPMIv2 spec */
+	int cipher_suite_id = 1; /* See table 22-19 of the IPMIv2 spec */
 	int argflag, i, found;
 	int rc = -1;
 	char sol_escape_char = SOL_ESCAPE_CHARACTER_DEFAULT;
