--- ../src/ipmievd.c~	2005-04-06 23:49:48.000000000 +0300
+++ ../src/ipmievd.c	2005-04-10 17:28:54.000000000 +0300
@@ -240,7 +240,7 @@
 
 static void usage(void)
 {
-	fprintf(stderr, "usage: ipmievd [-hvd]\n");
+	fprintf(stderr, "usage: ipmievd [-hvs]\n");
 	fprintf(stderr, "\n");
 	fprintf(stderr, "       -h        This help\n");
 	fprintf(stderr, "       -v        Verbose (can use multiple times)\n");
